packages/node-modules-inspector/src/app/components/display/PackageSpec.vue

```diff
@@ -17,7 +30,13 @@ const props = defineProps<{
         i-catppuccin-folder-packages-open icon-catppuccin mr2 absolute left-0 top-0
       />
     </span>
-    <DisplayPackageName :name="props.pkg.name" />
-    <DisplayVersion op50 :version="props.pkg.version" prefix="@" />
+    <DisplayPackageName :name="props.pkg.name" :pkg="props.pkg" />
+    <DisplayVersion
+      op50
+      :version="props.pkg.version"
+      prefix="@"
+      :class="{ 'text-red-500 !op100': isCurrentDeprecated }"
```

Comment:
I feel it's better to do a line-through with less saturated red.

Reply:
I did a tentative, but I'm not sure
![image](https://github.com/user-attachments/assets/9af9a95e-4981-444a-a3f2-c2843b1b35d2)

I'll leave it like this for now, feel free to tune it. I have close to 0 sense of design.